src/extractor/CaretValueRuleExtractor.ts

```diff
@@ -231,6 +231,42 @@ export class CaretValueRuleExtractor {
     return caretValueRules;
   }
 
+  static processConcept(
+    input: any,
+    conceptHierarchy: string[],
+    codeSystemName: string,
+    fisher: utils.Fishable
+  ): ExportableCaretValueRule[] {
+    const caretValueRules: ExportableCaretValueRule[] = [];
+    const flatConcept = getPathValuePairs(input);
+    Object.keys(flatConcept)
+      .filter(
+        key =>
+          !CONCEPT_IGNORED_PROPERTIES.some(
+            property => key === property || new RegExp(`${property}(\\[\\d+\\])?\\.`).test(key)
+          )
+      )
+      .forEach(key => {
+        const caretValueRule = new ExportableCaretValueRule('');
+        caretValueRule.caretPath = key;
+        caretValueRule.value = getFSHValue(key, flatConcept, 'Concept', fisher);
```

Comment:
I think here you could up front change this to 
```
        caretValueRule.value = getFSHValue(`concept.${key}`, flatConcept, 'CodeSystem', fisher);
```
And then you would not need to change the `getFSHValue` function at all I think?

Reply:
This is a clever idea, but doing it this way makes getting the value within the `flatConcept` [on the first line of the `getFSHValue` function](https://github.com/FHIR/GoFSH/blob/14582d5c0ace45370328cd50f15e6edaddee9ae4/src/utils/element.ts#L56) impossible with the key parameter. I could do some clever string manipulation in that function, or change the keys within the `flatConcept` object to also have `concept.` prefixes, but I thought that both of these options felt a bit less intuitive than what I'm currently doing in the `getFSHValue` function. If you prefer either of these approaches then I can make the change!

Reply:
Ah good point. How it is now is fine!